### PR TITLE
Add Eatery test to check certain fields always have a value

### PR DESCRIPTION
**Pull request by seancorc**
## Overview
Added two tests (a query for both campus and collegetown eateries) to check that the fields which should always have a value indeed do.

## Changes Made
Added two functions and Test objects to eatery/tests.py that get data from the eatery backend and check to make sure all fields have a value.

### CODE REVIEW

File: src/eatery/tests.py [hunk 1]
```diff
@@ -3,9 +3,85 @@
...
+    url
+  }
+}
+"""
+)
+
+
+def all_campus_fields_non_empty(r):
+    response = r.json()
+    campust_eateries = response["data"]["campusEateries"]
```
Comment (meganle) [inline src/eatery/tests.py]: `campus_eateries` instead?

File: src/eatery/tests.py [hunk 2]
```diff
@@ -3,9 +3,85 @@
...
+      latitude
+      longitude
+    }
+    eateryType
+    paymentMethodsEnums
+    phone
+    address
+    categories
+    price
+		ratingEnum
```
Comment (meganle) [inline src/eatery/tests.py]: indentation?

File: src/eatery/tests.py [hunk 3]
```diff
@@ -3,9 +3,85 @@
...
+    # Iterate over fields
+    for field in campust_eateries:
+        if field is None:
+            return False
+    return True
+
+
+def all_collegetown_fields_non_empty(r):
+    response = r.json()
+    campust_eateries = response["data"]["collegetownEateries"]
```
Comment (meganle) [inline src/eatery/tests.py]: same as above?

**Review by meganle — CHANGES_REQUESTED** [review]

File: src/eatery/tests.py [hunk 4]
```diff
@@ -3,9 +3,85 @@
...
+      latitude
+      longitude
+    }
+    location
+    paymentMethodsEnums
+    campusArea{
+      descriptionShort
+    }
+    location
+    slug
```
Comment (young-k) [inline src/eatery/tests.py:24]: minor, but prefer alphabetizing all of these fields

File: src/eatery/tests.py [hunk 5]
```diff
@@ -3,9 +3,85 @@
...
+        if field is None:
+            return False
+    return True
+
+
+def all_collegetown_fields_non_empty(r):
+    response = r.json()
+    campust_eateries = response["data"]["collegetownEateries"]
+    # Iterate over fields
+    for field in campust_eateries:
```
Comment (young-k) [inline src/eatery/tests.py]: `return campus_eateries.all()` should suffice

File: src/eatery/tests.py [hunk 6]
```diff
@@ -10,18 +10,17 @@
     """{
   campusEateries {
     about
-    name
-    nameShort
+    campusArea{
```
Comment (kq-li) [inline src/eatery/tests.py]: space before brace

**Review by kq-li — APPROVED** [review]
lgtm!

**Review by meganle — APPROVED** [review]
nice job!

**Pull request merged**